proposed/2021/VisualStudioDeepLinkNuGetPackageInstall.md

```diff
@@ -45,6 +45,10 @@ Since this feature requires a minimum version of Visual Studio to function, the
 
 Depending on how the URI link is set up, that link will be split up into its key components that provide me with the information needed to describe a package. Each component will be sub stringed analyzed, and an object will be created with fields associated with said components such as package name and version number. This way, the information in the URI can live somewhere without having to expose the URI in places that it doesn&#39;t need to be.
 
+##### URI Format
+
+To provide context for the point above, the url will be formatted as follows: **vsph://OpenPackageDetails/<packageName>/<version>**, where *packageName* is the name of the NuGet package and *version* is the version number of the target NuGet package. 
```

Comment:
Visual Studio supports not only .NET projects, but also NodeJS, Python, and I believe there's a 3rd party Rust project system. All of these languages have official package managers.

Is your intention to make this URL format suitable for all of them to use? If so, I think this spec needs more details about how project systems and package managers for other languages should use this extensibility, how language selection will work for project types that NuGet doesn't support, etc.

Otherwise, the URL format should be clearly specific to NuGet.

Reply:
Also, does Visual Studio mandate the `vsph://` scheme?